composer.json

```
@@ -30,7 +30,8 @@
         "symfony/filesystem": "~3.0|~4.0",
         "symfony/yaml": "~3.0|~4.0",
         "symfony/process": "~3.0|~4.0",
-        "phpdocumentor/reflection-docblock": "~2.0"
+        "phpdocumentor/reflection-docblock": "~2.0",
+        "phpmyadmin/twig-i18n-extension": "^3.0"
```

```suggestion
        "wdes/php-i18n-l10n": "^2.0"
```
https://github.com/wdes/php-I18n-L10n